content/versions-comparison/2.0-news.json

```diff
@@ -21,7 +21,7 @@
     "link": "https://api.akeneo.com/api-reference-index.html#Familyvariant"
   },
   {
-    "title":"User group permissions on CSV import and export 🔒",
+    "title":"Additional user permissions on native imports/exports 🔒",
```

Comment:
Why the use  of "native" here? Feels like it doesn't add any info.
On another note, this title is maybe too long. Did you check the display?